src/tools/rbd/action/Device.cc

```diff
@@ -196,10 +197,8 @@ void get_unmap_arguments(po::options_description *positional,
     ("image-or-snap-or-device-spec",
      "image, snapshot, or device specification\n"
      "[<pool-name>/[<namespace>/]]<image-name>[@<snap-name>] or <device-path>");
-  at::add_pool_option(options, at::ARGUMENT_MODIFIER_NONE);
-  at::add_namespace_option(options, at::ARGUMENT_MODIFIER_NONE);
-  at::add_image_option(options, at::ARGUMENT_MODIFIER_NONE);
-  at::add_snap_option(options, at::ARGUMENT_MODIFIER_NONE);
+  at::add_snap_spec_options(positional, options, at::ARGUMENT_MODIFIER_NONE);
```

Comment:
This is wrong: `at::add_snap_spec_options` doesn't mention `or <device-path>` in the description -- that is why it's open-coded here.